src/main/java/com/mail/entities/TextMessage.java

```diff
@@ -0,0 +1,34 @@
+package com.mail.entities;
+
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+import org.springframework.lang.NonNull;
+
+import java.time.LocalDate;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class TextMessage {
+
+    int id;
+
+    String title;
+
+    @NonNull
```

Comment:
If you wish to validate endpoint parameters, you should use `javax.validation.constraints.NotNull`.
It is a standart validation that can be supported by IDE, while annotations from Spring should be supported by explicitly by IDE (via plugin for example).

You can switch to other validators in case of more complex validations when `javax.validation` package is lacking needed features.